docs/index.html

```diff
@@ -0,0 +1,201 @@
+
```

Comment:
You can publish from a directory in the master branch now. :)

https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/